packages/core/nuxt-module/lib/plugins/i18n-cookies.js

```diff
@@ -27,8 +27,13 @@ const i18nCookiesPlugin = ({ $cookies }) => {
     expires: new Date(new Date().setFullYear(new Date().getFullYear() + 1)) // Year from now
   };
 
-  !$cookies.get(VSF_CURRENCY_COOKIE) && $cookies.set(VSF_CURRENCY_COOKIE, settings.currency, cookieOptions);
-  !$cookies.get(VSF_COUNTRY_COOKIE) && $cookies.set(VSF_COUNTRY_COOKIE, settings.country, cookieOptions);
+  const cookieNames = {
+    currency: i18n.cookies?.currencyCookieName || VSF_CURRENCY_COOKIE,
```

Comment:
If it's in `cookies` key, maybe just `currency` and `country` names would be better, will be simpler. WDYT?

Reply:
@dawid-ziobro I agree, but we already have those keys defined and used in such way in our ct api-client (commercetools/api-client/src/index.server.ts):

`const getI18nConfig = (req, configuration) => {
  const cookieSettings = configuration.cookies || defaultSettings.cookies;
  const { currencyCookieName, countryCookieName, localeCookieName, storeCookieName } = cookieSettings;
  const locale = req.cookies[localeCookieName] || configuration.locale || defaultSettings.locale;
  const currency = req.cookies[currencyCookieName] || configuration.currency || defaultSettings.currency;
  const country = req.cookies[countryCookieName] || configuration.country || defaultSettings.country;
  const store = req.cookies[storeCookieName] || configuration.store;

  return { currency, country, locale, store };
};`

So for the sake of consistency I'd stick with what we have already defined in other places.

Reply:
Ah ok ;)